README.md

```diff
@@ -91,8 +122,8 @@ ourBoard.autoFindOpenBCIBoard(function(portName,ports) {
 Running
 -------
 1. Plug usb module into serial port
-2. Turn OpenBCI device on
-3. Type `npm start` into the terminal in the project directory
+1. Turn OpenBCI device on
+1. Type `npm start` into the terminal in the project directory
```

Comment:
I thought github convention is to put a 1 in front of every list item. When i try to do 2 it does not work